tools/mkimage-raw-bios/Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: push forcepush tag forcetag show-tag
+
```

Comment:
needs removing now

Reply:
Why?

Reply:
I think all the Makefiles went in Ian's update?

Reply:
There was a discussion about it. In the end, came down to philosophy. I like them as an "entrypoint" or "wrapper" that makes it obvious what to do and simplifies it. I still find it easier to do `make push` and `make tag` than `lkt pkg push .`

Actually, I took the exact contents of the `Makefile` from one of them in there.

Doesn't hurt, does it?

Reply:
Either all packages should have it or none should and certainly we shouldn't be reproducing a dozen lines of boilerplate here and in your other PR, it should be a single line include.

Reply:
> Either all packages should have it or none should

I like make and a global entrypoint. But I get that it is a personal preference.

> it should be a single line include

I don't understand? Single line include?

Reply:
`include ../../../something.mk` which contains the lines you are currently duplicating all over the place.

Reply:
oh right. Let me see if that works. You did put a `Makefile` in `tools/Makefile`, although that has logic to hit all directories below.

Reply:
Although ased on discussions on the slack channel where the existing `pkg/Makefile` has already caused confusion about how to override options I don't think I'm in favour of propagating that to all the indivdual packages -- otherwise the docs have to be "`run make «foo», unless you want to pass another options, in which case run `linuxkit pkg foo --option`". I don't think splitting our instructions like that is wise, they should just talk about how to run `linuxkit`. If there are `Makefiles` then people will expect to use them even if they need something other than the default.

So having thought about it I'm pretty against having needless Makefiles.

Reply:
Well, including `../Makefile` sort of works. Sort of.

Let's not get into the deeper philosophical discussion here. I prefer `Makefile` even (or especially) when they wrap saner tools (like `lkt pkg ...`).  I have them for every one of my `go` and `node` and whatever projects, even if `go build .` or `npm start` is painfully obvious. But I don't care enough to hold these PRs up.

@ijc and @justincormack do we want to remove the `Makefile` here and the other PRs and move ahead?

Reply:
> You did put a Makefile in tools/Makefile, although that has logic to hit all directories below.

Right that is just for the iteration and as I said I'm now realising it wasn't a very good idea because it has confused at least one person so far.

I'm thinking about other more flexible ways that could be achieved. I don't think I want `linuxkit bulk-pkg build «dir1» «dir2» .. «dirN»`, but perhaps I'll think of something better.

> do we want to remove the Makefile here and the other PRs and move ahead?

Yes, please.

Reply:
Done in all 3 PRs.